packages/@okta/vuepress-site/code/angular/okta_angular_sign-in_widget/index.md

```diff
@@ -20,7 +20,8 @@ If you do not already have a  **Developer Edition Account**, you can create one
 
 ## Add an OpenID Connect Client
 
-* Sign in to the Okta Developer Dashboard, and select **Create New App**
+* Sign in to the Okta Developer Dashboard, and select **Create New App**. <br/>
+You can sign in to Okta Developer Dashboard through [https://login.okta.com](https://login.okta.com) and clicking `Admin` button on Okta Dashboard or by signing in directly into Okta Developer Dashboard through URL which looks like [hxxps://dev-1235813-admin.okta.com](hxxps://dev-1235813-admin.okta.com).
```

Comment:
@albertchen-okta confirm please, but I don't think we want to have any instructions for the Developer Console?